src/ort_genai_c.cpp

```diff
@@ -7,6 +7,7 @@
 #include "span.h"
 #include "ort_genai_c.h"
 #include "generators.h"
+#include "logits_processor.h"
```

Comment:
Is this needed?

Reply:
Yeah, Cause it now use same forward declaration as search, model. We need to include their full definition to build with unique_ptr<LogitsProcessor> in generators.h